src/test/java/com/github/fishio/audio/AudioTestUtil.java

```diff
@@ -36,7 +36,7 @@ public static boolean supportsMusic() {
 	public static boolean supportsSoundEffects() {
 		try {
 			Path path = AudioUtil.getAudioFiles(true).get(0);
-			new SoundEffect(path.toUri().toString());
+			new SoundEffect(path.toUri().toString()).isPlaying();
```

Comment:
Don't you think this is a cheaty way to fix the warning?
I think adding .isPlaying() behind it just makes it more confusing.
There is no need to fix every warning, especially when it's wrong.
(plus my Eclipse doesn't even complain about instansiating an object without storing it in a variable).

<layout